src/desktop/apps/artwork/components/metadata/components/artwork_classification/index.coffee

```diff
@@ -0,0 +1,74 @@
+modalize = require '../../../../../../components/modalize/index.coffee'
+
+module.exports = ->
+  attributionClassDetailsHtml = """
+    <div class='attribution-class-container'>
```

Comment:
I wonder if it's cleaner to put the markup in a separate Jade template and require it here, like [this example](https://github.com/artsy/force/blob/1df4030b1f4e1ed730f950700f645aebc2f8e1fb/src/desktop/components/feedback_modal/views/press.coffee#L2).